nornir/plugins/tasks/networking/napalm_rollback.py

```diff
@@ -0,0 +1,16 @@
+from nornir.core.task import Result, Task
+
+
+def napalm_rollback(task: Task, rollback: bool = False) -> Result:
+    """
+    Rollback device configuration using napalm
+
+    Arguments:
+        rollback: whether to rollback the configuration or not
```

Comment:
Instead of a `rollback` parameter it'd be nice to have a `dry_run` instead, pretty much following the same pattern as `napalm_configure`: https://github.com/nornir-automation/nornir/blob/fee75ff2902e1a6af72a6981b2dbbff9be7e290d/nornir/plugins/tasks/networking/napalm_configure.py

Reply:
OK, will do. Any preference on what the default should be for `dry_run`? I'm thinking `True` since false won't doesn't provide any feedback to the user on what changes will be rolled back.

Reply:
Default should be `None` like with `napalm_configure`.

Reply:
Ok, all set.